pkg/pow/pool.go

```diff
@@ -159,7 +162,7 @@ func (p *StratumPool) GetWork(input chan *Work) (*Work, bool) {
 	return nil, false
 }
 
-func (p *StratumPool) Submit(ctx context.Context, result *Result) bool {
+func (p *StratumPool) Submit(ctx context.Context, result *Result) (*types.Transaction, error) {
```

Comment:
not sure what this method actually does since it doesn' submit anything to the tellor contract.